packages/ui/src/config/ConfigContext.tsx

```diff
@@ -30,6 +30,23 @@ interface ConfigContextState {
   showActivityIndicator: boolean;
   showSubtitles: boolean;
   headerHeight: number;
+  blockInteractionConfig: {
+    visible: boolean;
+    message: string;
+    iconName: string;
+    callback: () => void;
+  };
```

Comment:
I think you defined this type like three or four times, do not hesitate to create real interface and use this interface in the rest of the code 
```
 interface BlockInteractionConfig {
    visible: boolean;
    message: string;
    iconName: string;
    callback: () => void;
  };
```